docs/guides/3-javascript.md

```diff
@@ -45,55 +56,87 @@ spectral.setRuleset({
     },
   },
 });
+
+// we lint our document using the ruleset we passed to the Spectral object
 spectral.run(myDocument).then(console.log);
 ```
 
-This will run Spectral with no formats, rules or functions, so it's not going to do anything besides \$ref resolving.
-Find out how to add formats, rules and functions below.
+## Load Rulesets and API Specification Files
 
-## Loading Rulesets
+Let's look at some other examples and how to work with external files.
 
-Spectral comes with some rulesets that are very specific to OpenAPI v2/v3, and they can be set using `Spectral.setRuleset()`.
+### Load a JSON/YAML Ruleset
 
-```js
-const { Spectral } = require("@stoplight/spectral-core");
-const ruleset = require("./my-ruleset"); // this works only for JS ruleset, look at the section below to learn how to load a YAML/JSON ruleset
+If you would like to run this example, make sure that you have:
 
-const spectral = new Spectral();
-spectral.setRuleset(ruleset);
-// lint
+- An OpenAPI specification file in the same directory as your script named `petstore.yaml`. You can use the one found [here](https://github.com/OAI/OpenAPI-Specification/blob/main/examples/v3.0/petstore.yaml).
```

Comment:
```suggestion
- An OpenAPI description document in the same directory as your script named `petstore.yaml`. You can use the one found [here](https://github.com/OAI/OpenAPI-Specification/blob/main/examples/v3.0/petstore.yaml).
```

Can we use literally any other example than the petstore? It's a really bad spec and its very associated with old Swagger tooling. 

There's a bunch in here you could use: https://github.com/stoplightio/public-apis

Reply:
I thought people were still using petstore 😅 . I updated it with the Plaid API description from the repository you linked to. :)

Reply:
Some people are but we don't have to perpetrate that. 😇